### ISSUE TITLE
join the game crashes game

### ISSUE THREAD

**Issue by MrMiYaGii** [LWSS/McDota#24]
when i inject i can open menu but when i join game game crashes im kubuntu 19.04
kernel 5.0.0 i cmake and make with no issues

**Comment by LWSS** [comment]
ok well the way that works, it that I signal a flag on the entity and it triggers the Valve-made debug boxes. 

Did you change your resolution or something?

**Comment by LWSS** [comment]
Also, send pic, some of the heroes are really sloppy

**Comment by MrMiYaGii** [comment]
yeah i changed resolution but 1 anoying bug or sometihng that u need to inject cheat while u in game otherwise gamecrashes if u inject in menu and start game and sometimes it just crash middle of match

**Comment by kolay-v** [comment]
> yeah i changed resolution but 1 anoying bug or sometihng that u need to inject cheat while u in game otherwise gamecrashes if u inject in menu and start game and sometimes it just crash middle of match

Same

Alse not only boxes, but all visuals work very bad. They have big offset from entities

**Comment by LWSS** [comment]
> > yeah i changed resolution but 1 anoying bug or sometihng that u need to inject cheat while u in game otherwise gamecrashes if u inject in menu and start game and sometimes it just crash middle of match
> 
> Same
> 
> Alse not only boxes, but all visuals work very bad. They have big offset from entities

Literally not my fault, it's how valve modeled them lol! You should see the tiny one

**Comment by kolay-v** [comment]
https://i.imgur.com/4tyGExA.png

**Comment by LWSS** [comment]
something to do with your resolution, it being changed mid-game or your window manager
